### ISSUE TITLE
Support zero sum DKP

### ISSUE THREAD

**Issue by SamMousa** [ClassicLootManager/ClassicLootManager#41]
Zerosum DKP was requested via discord.

We have some settings for it, but no implementing code yet.

**Additional context**
Requested by Rashaka from [Critical Mass] on [Atiesh]

**Comment by SamMousa** [comment]
@lantisnt my idea for implementing this:

- [ ] Use different kind of award entry (since we need to have all players that are credited in there)
- [ ] Add new mutator that decreases winner's DKP and awards that DKP to the rest
- [ ] Figure out rounding, how much DKP disappearing is acceptable?
- [ ] What is the inflation setting?

Assumptions:
- The raid itself is not in the ledger, therefore we must store all players with the event.
- Enabling / disabling zero sum (even in the past) does not affect existing rewards.

Possible improvements:
- If we store the raid in the ledger we can track who's actually there (attendance records)
- Storing the raid once is cheaper than doing it for every item that is awarded.

**Comment by lantisnt** [comment]
Postpone until #43 is done and use the fact of tracking players

**Comment by lantisnt** [comment]
#43 Finished.
Resolves #19

**Comment by SamMousa** [comment]
Blocked by #59
